code/modules/mob/living/silicon/mommi/inventory.dm

```diff
@@ -27,6 +27,12 @@
 	// And before you ask, this is how /mob handles NULLs, too.
 	if(!W)
 		return 0
+	if(cell && cell.charge <= 100)
```

Comment:
Well no, that would make no sense.
The change to lowpower wouldn't come based on the size of the cell.
It would also break if you made an MoMMI with an infinite cell.